manual/core/integration/README.md

```diff
@@ -8,6 +8,73 @@
 
 -----
 
+### Which artifact(s) should I use?
+
+There are multiple driver artifacts under the group id
+[com.datastax.oss](https://search.maven.org/search?q=g:com.datastax.oss). Here's how to pick the
+right dependencies:
+
+<table>
+<tr><th>Feature</th><th>Artifact(s)</th><th>Comments</th></tr>
+<tr>
+  <td>
+    Core functionality: executing queries with <code>CqlSession.execute()</code>, processing the
+    results with <code>ResultSet</code>, etc.
+  </td>
+  <td><code>java-driver-core</code></td>
+  <td></td>
+</tr>
+<tr>
+  <td>
+    Same as the above, but without an explicit dependency to Netty. 
```

Comment:
It currently also shades ESRI and Jackson, both legacy and new. Maybe mention Jackson as well, since it's a popular library?